lib/zipkin-tracer/config.rb

```diff
@@ -62,6 +66,15 @@ def adapter
       sampled_as_boolean: true
     }
 
+    def parse_record_on_server_receive(tag_names)
```

Comment:
Not sure I get it, here I'm parsing the value passed in the config once and for all, just like the other config options. I don't think that parsing whatever was passed at config time belongs in the rack class since it has nothing to do with middleware stuff.